src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -3581,9 +3581,9 @@ private BoundExpression BindConditionalOperator(ConditionalExpressionSyntax node
                     trueExpr,
                     falseExpr,
                     this.Conversions,
-                    this.Compilation.IsFeatureEnabled(MessageID.IDS_FeatureStaticNullChecking),
-                    out hadMultipleCandidates,
-                    ref useSiteDiagnostics);
+                    includeNullability: false,
```

Comment:
Was there a reason why a couple of places in initial binding changed (no longer look for flag) but most didn't? Did those cause trouble?
If not, maybe leave them until ready to remove across the board from initial binding?

Reply:
I stopped short of changing all binding to use `includeNullability: false` because there were already many changes to `NullableWalker` and I didn't want to introduce more. That said, we should change all binding to `includeNullability: false` so I'd like to leave the changes that have been made so far.